integration/okteto/deploy_test.go

```diff
@@ -145,6 +145,73 @@ func TestDeploySuccessOutput(t *testing.T) {
 	}
 }
 
+func TestDeployWithNameSuccessOutput(t *testing.T) {
```

Comment:
```suggestion
func TestDeployWithNonSanitizedNameSuccess(t *testing.T) {
```

As mentioned in other comment, I think we should remove the output checks. There should be other tests for it